platform-configs/include/protectli-common.robot

```diff
@@ -80,6 +80,10 @@ ${SECURE_BOOT_SUPPORT}=                         ${TRUE}
 ${SECURE_BOOT_DEFAULT_STATE}=                   Disabled
 ${USB_STACK_SUPPORT}=                           ${TRUE}
 ${USB_MASS_STORAGE_SUPPORT}=                    ${TRUE}
+${UEFI_PASSWORD_SUPPORT}=                       ${TRUE}
```

Comment:
Password is in Vault Pro (VP series), but not in all (e.g. V1000 series). It should go to protectli-pro. And already is there.

Reply:
I don't think it makes sense since half of the VP-specific options need to be disabled anyway.

Reply:
Then please implement what makes sense and works. Definitely we cannot include flags such as password in common, since these are not common for all boards.

Reply:
I didn't mean to push that change again, it was unnoticed.

Reply:
@pkubaj 

> I didn't mean to push that change again, it was unnoticed.

But it is still there, after two weeks of this discussion :disappointed:

Reply:
I have just fixed it myself